packages/styleguide/src/examples/pages/aui-toastr/aui-toastr.page.ts

```diff
@@ -0,0 +1,45 @@
+import {Component} from '@angular/core';
+import {ToastrService} from 'ngx-toastr';
+
+@Component({
+  selector: 'app-toastr-demo',
+  templateUrl: './aui-toastr.page.html'
+})
+export class ToastrDemoPageComponent {
+  importModule = `import { AuiToastrModule } from '@acpaas-ui/ngx-toastr';
+
+@NgModule({
+  imports: [ AuiToastrModule ]
+})
+export class AppModule { }
+  `;
```

Comment:
Waarvoor dient deze regel?

Reply:
Dat is demo code die op de examples page tevoorschijn komt

Reply:
Ah zo, nu zie ik het pas. Prima!